src/functionaltests/java/com/ericsson/ei/subscriptions/trigger/SubscriptionTriggerSteps.java

```diff
@@ -122,8 +123,9 @@ public void subscriptions_are_setup_using_REST_API(String endPoint) throws Throw
     @When("^I send Eiffel events$")
     public void send_eiffel_events() throws Throwable {
         LOGGER.debug("About to send Eiffel events.");
-        List<String> notSentEvents = sendEiffelEvents(EIFFEL_EVENTS_JSON_PATH);
-        assertEquals("[]", notSentEvents.toString());
+        List<String> eventsIdList = sendEiffelEvents(EIFFEL_EVENTS_JSON_PATH);
+        List<String> missingEvents = getMissingEvents(eventsIdList);
+        assert(missingEvents.size() == 0) : "The following events are missing in mongoDB: " + missingEvents.toString();
```

Comment:
Why not use standard asserEquals with failure messages as we do in other test.
"assertEquals(java.lang.String message, java.lang.Object expected, java.lang.Object actual) "